core/server/data/migrations/versions/3.38/06-add-newsletter-settings.js

```diff
@@ -0,0 +1,18 @@
+const logging = require('../../../../../shared/logging');
+
+module.exports = {
```

Comment:
Could/should we try and use `createTransactionalMigration` here?

Reply:
Done!